src/containers/Configure/Configure.js

```diff
@@ -0,0 +1,89 @@
+import React, { useState } from 'react'
+import { Switch, Route, Redirect } from "react-router-dom";
+import './Configure.css';
+
+const Configure = (props) => {
+  const [systemSize, updateSystemSize] = useState('')
+  const [moduleType, updateModuleType] = useState('')
+  const [arrayType, updateArrayType] = useState('')
+  const [systemLosses, updateSystemLosses] = useState('')
+  const [tilt, updateTilt] = useState('')
+  const [azimuth, updateAzimuth] = useState('')
+
+  return (
+    <section className="configure-container">
+      <h2>Enter in the following information to get a quote!</h2>
+      <form>
+        <h3>Solar Array Configuration</h3>
+        <div className="label-input-container">
+          <label>System Size (kW):</label>
+          <input
+            type="number"
+            placeholder="4"
+            value={systemSize}
+            required
+            onChange={e => updateSystemSize(e.target.value)}
+          />
+        </div>
+        <div className="label-input-container">
+          <label>Module Type:</label>
+          <select
+            className="configure-selects"
+            onChange={e => updateModuleType(e.target.value)}
+          >
+            <option>Select..</option>
+            <option>Standard</option>
+            <option>Premium</option>
+            <option>Thin Film</option>
+          </select>
+        </div>
+        <div className="label-input-container">
+          <label>Array Type:</label>
+          <select
+            className="configure-selects"
+            onChange={e => updateArrayType(e.target.value)}
+          >
+            <option>Select..</option>
+            <option>Fixed (open rack)</option>
+            <option>Fixed (roof mount)</option>
+            <option>1-Axis Tracking</option>
+            <option>1-Axis Backtracking</option>
+            <option>2-Axis Tracking</option>
```

Comment:
Looking at the NREL API documentation again, it looks like these options will need to map out to integer values 0-4 for the API to accept.

Reply:
Good pick up.  We will make the adjustments when we write the handle submit function for this page